docs/docs/deploying-to-gatsby-cloud.md

```diff
@@ -0,0 +1,76 @@
+- **Cloud Quickstart** that empowers new and non-technical users to create projects in minutes with a Content Management System (CMS), sample content, and connected Gatsby starter.
+- **Real-time Preview** to simplify content creation and collaboration. Preview offers a private playground for developers, designers, marketers, and content creators by providing a shareable temporary URL for viewing changes immediately and in context. With instant updates triggered by a CMS or webhooks, it’s basically a shareable, hot-reloading preview.
+- **CMS Integrations** that are unrivaled in the market
+- **Builds** is the fastest continuous deployment solution for Gatsby sites and apps- up to 20x faster build times compared to other solutions. Build with Gatsby and deploy to your favorite CDN.
+- **Reports** provide automated Lighthouse performance checks and deploy previews to fix errors before they’re published. 
+
+Best of all, Gatsby Cloud includes a [free tier](https://www.gatsbyjs.com/pricing/) designed to comfortably support personal and small sites. 
+
+## Integrations
+Gatsby Cloud integrates with the tools you already use to build sites. By connecting your Gatsby project's Github repo, Gatsby Cloud automatically builds and deploys your site when you make changes.
+
+### CMS Integrations
+Gatsby Cloud offers integrations with a wide variety of CMSs. The below CMSs have first-class, automatic integrations with Gatsby Cloud: 
+- Contentful
+- Cosmic JS
+- Dato CMS
+- Sanity.io
+
+If you want to work with a CMS without automatic integration support you still can. There are specific documents available for the below integration points:
+- Contentstack
+- Drupal
+- Strapi
+
+In addition, Gatsby Cloud offers a POST endpoint for manually integrating with CMSs that support webhooks.
+
+### Hosting Integrations
+Gatsby Cloud offers automatic integration with the following hosting providers:
+ - Netlify
+ - Amazon S3
+ - Firebase
+ - Google Cloud Storage
+ - Fastly
+
+Please refer to the [Gatsby Cloud Docs](https://www.gatsbyjs.com/docs/) for full details on available integrations.
+
+## Setup a new Gatsby site from scratch
+1. Head over to [Gatsby Cloud](https://www.gatsbyjs.com/get-started/) and sign-up/sign-in with your Github account if you haven't already. 
+2. Navigate to the [get started](https://www.gatsbyjs.com/get-started/) page or click the **Create new site** button on your [Dashboard](https://www.gatsbyjs.com/dashboard/sites).
+3. Choose the **I don't have a Gatsby site yet** option.
+4. On *Tab 1* choose from the Starter options and click **Next**.
+5. On *Tab 2*, enter a name for your new project and click **Next**. This will be the name of the project repo added to your Github account. You will need to set your [github permissions](https://github.com/settings/installations) to enable "All Repositories" access to allow Gatsby Cloud to create a new repo.  
```

Comment:
```suggestion
5. On *Tab 2*, enter a name for your new project and click **Next**. This will be the name of the project repo added to your GitHub account. Set your [GitHub permissions](https://github.com/settings/installations) to enable "All Repositories" access in order to allow Gatsby Cloud to create a new repo.  
```